src/routes/profile/profile.route.test.ts

```diff
@@ -0,0 +1,81 @@
+import { startServer } from '../../app'
+import type { Express } from 'express'
+import supertest from 'supertest'
+import { dataSource } from '../../configs/dbConfig'
+
+const randomString = Math.random().toString(36)
+let server: Express
+let accessToken: string
+
+describe('profile', () => {
+  beforeAll(async () => {
+    server = await startServer()
+
+    const testUser = {
+      email: `test${randomString}@gmail.com`,
+      password: '123'
+    }
+
+    await supertest(server)
+      .post('/api/auth/register')
+      .send(testUser)
+      .expect(201)
+
+    const response = await supertest(server)
+      .post('/api/auth/login')
+      .send(testUser)
+      .expect(200)
+
+    accessToken = response.body.token
+  }, 5000)
+
+  describe('Get profile route', () => {
+    it('should return a 401 without a valid access token', async () => {
```

Comment:
```suggestion
    it('should return a 401 when a valid access token is not provided', async () => {
```

Reply:
Shall we use a similar format with other tests as well?

Reply:
yeah sure